archesproject/arches
# 6687 json csv import

### Types of changes


### Description of Change

fixes #6687 

### Issues Solved

#6687 



### To Test
load usd ssc, ensure celery is running, and run through Initiate Fiscal Year workflow, upload attached file, search for the parcel instances and ensure that an owner tile with the correct related Fiscal Year is saved, as well as unit instances with inspection tiles saved with the correct related Fiscal Year (this is despite the data for related Fiscal Year not being defined/cast as json or string)

[nbs_parcels.xlsx](https://github.com/archesproject/arches/files/5344486/nbs_parcels.xlsx)


### Checklist


#### Ticket Background
*   Sponsored by: 
*   Found by: @ 
*   Tested by: @ 
*   Designed by: @ 

### Further comments
